frontend/src/pages/GeoLocationPage.vue

```diff
@@ -17,64 +38,98 @@
                     {
-                        country: 'World',
-                        visitors: '175',
-                        new_visitors: '50',
-                        sessions: '331',
-                        bounce_rate: '22.5%',
-                        avg_session_time: '00:32:45'
+                        country: 'US',
+                        visitors: '1',
+                        new_visitors: '12',
+                        sessions: '10',
+                        bounce_rate: '88',
+                        avg_session_time: '00:30:00'
                     },
                     {
-                        country: 'USA',
-                        visitors: '25',
+                        country: 'Canada',
+                        visitors: '48',
                         new_visitors: '12',
-                        sessions: '45',
-                        bounce_rate: '14%',
+                        sessions: '67',
+                        bounce_rate: '14',
                         avg_session_time: '00:30:00'
                     },
                     {
                         country: 'Ukraine',
                         visitors: '32',
                         new_visitors: '10',
-                        sessions: '65',
-                        bounce_rate: '21%',
+                        sessions: '32',
+                        bounce_rate: '21',
                         avg_session_time: '00:37:00'
                     },
                     {
-                        country: 'Canada',
+                        country: 'Brazil',
                         visitors: '87',
                         new_visitors: '23',
                         sessions: '175',
-                        bounce_rate: '5%',
+                        bounce_rate: '45',
+                        avg_session_time: '00:43:00'
+                    },
+                    {
+                        country: 'Germany',
+                        visitors: '65',
+                        new_visitors: '23',
+                        sessions: '10',
+                        bounce_rate: '5',
                         avg_session_time: '00:43:00'
                     },
                     {
                         country: 'Japan',
                         visitors: '31',
                         new_visitors: '5',
                         sessions: '46',
-                        bounce_rate: '49%',
+                        bounce_rate: '12',
                         avg_session_time: '00:21:00'
                     },
                 ]
             };
+        },
+        computed: {
+            ...mapGetters('geo_location', {
+                getSelectedPeriod: GET_SELECTED_PERIOD,
+                getSelectedParameter: GET_SELECTED_PARAMETER,
+            }),
+        },
+        methods: {
+            ...mapActions('geo_location', {
+                changeSelectedPeriod: CHANGE_SELECTED_PERIOD
+            }),
+            changePeriod(data) {
```

Comment:
If you need to do anything before action is called, this method is ok, but if not - you can just use `changeSelectedPeriod` inside template and remove this method